SwiftUI_DZ/SwiftUI_DZ/View/PriceListView.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  PriceListView.swift
+//  SwiftUI_DZ
+//
+//  Created by Ernest Avagovich on 20.01.2023.
+//
+
+import SwiftUI
+
+/// Экран выбора тарифа
+struct PriceListView: View {
+    // MARK: - Private Constants
+    private enum Constants {
+        static let addToBasketText = "Добавть в корзину"
+        static let addToBasketImageName = "cart.fill.badge.plus"
+        static let subscriptionPlusText = "Оформите подписки 'Plus' для защиты своих данных"
+        static let noteImageName = "note"
+        static let sevenDayText = "7"
+        static let monthText = "30"
+        static let halfYearText = "182"
+        static let oneYearText = "365"
+        static let subscriptionSevenDayText = "Подписка на 7 дней за 99 руб."
+        static let subscriptionMonthText = "Подписка на 30 дней за 299 руб."
+        static let subscriptionHalfYearText = "Подписка на 182 дней за 599 руб."
+        static let subscriptionOneYearText = "Подписка на 365 дней за 999 руб."
+        static let buttonCornerRadius: CGFloat = 10
+        static let circleFrame: CGFloat = 100
+        static let vstackSpacing: CGFloat = 20
+        static let basketTagCount = 2
+    }
+    
+    // MARK: - Public Properties
+    @Binding var viewModel: VPNViewModel
```

Comment:
```suggestion
    @Binding var viewModel: VPNViewModel
```
Ее можно сделать @EnvironmentObject и заприватить, впрочем смотри сам)